src/Symfony/Component/Form/Extension/Core/Type/RepeatedType.php

```diff
@@ -31,13 +31,24 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             $options['options']['error_bubbling'] = $options['error_bubbling'];
         }
 
+        // children fields must always be mapped
+        $defaultOptions = ['mapped' => true];
+
         $builder
             ->addViewTransformer(new ValueToDuplicatesTransformer([
                 $options['first_name'],
                 $options['second_name'],
             ]))
-            ->add($options['first_name'], $options['type'], array_merge($options['options'], $options['first_options']))
-            ->add($options['second_name'], $options['type'], array_merge($options['options'], $options['second_options']))
+            ->add(
```

Comment:
Please keep in on one line to follow our standards, this also reduces the diff.

Reply:
Thought fabbot will complain about 120 symbols line.